gravitee-apim-rest-api/gravitee-apim-rest-api-model/src/main/java/io/gravitee/rest/api/model/api/ApiQuery.java

```diff
@@ -33,6 +34,7 @@ public class ApiQuery {
     private String label;
     private String state;
     private Visibility visibility;
+    private ExecutionMode executionMode;
```

Comment:
Is there a utility to search by `executionMode` ? I'm not sure we have identify any use case yet. Do you think it is really necessary ?